### ISSUE TITLE
Contact Us (page)

### ISSUE THREAD

**Issue by bryanwillis** [MBIStaffing/website#3]
## Todo For Live

- Which page do we want? The above or this ( http://mbistaffing.com/contact-old/). If we're using the first one we need to add text or buttons somewhere telling them that if they are applying for a job or want to hire they want to fill out a different form on the other pages. We talked about this, but I'm not positive on what we want to do here.
- Is info all accurate (address/phone/etc)?
- Add form descriptions / headers?
- Add live chat?

<img width="1238" alt="screen shot copy" src="https://cloud.githubusercontent.com/assets/4042621/19250707/7152ff6e-8f01-11e6-87f2-15b34e84a9ab.png">

**Comment by bryanwillis** [comment]
**[Thank You Page Redirect for Contact Forms](http://mbistaffing.com/contact/thank-you/)**
- We can pass any of the fields in the forms to the thank you pages, emails, etc. 

<img width="1129" alt="screen shot 37" src="https://cloud.githubusercontent.com/assets/4042621/19486675/f325622e-9524-11e6-9926-e7e5c42ce249.png">

**Comment by bryanwillis** [comment]
I think this page is good to go as long as the forms are right. I'm talking to Akken to figure out how to implement them into API

**Comment by bryanwillis** [comment]
**Under General Inquiry:**
@joedmbi  Do we want it to say "What are your current _IT staffing_ challenges" instead of just IT challenges?

**Comment by joedmbi** [comment]
Change phone number on contact us.  That is Tom's cell.

Joe Dix
MBi Staffing
(815) 880-5221

________________________________

**Comment by joedmbi** [comment]
Yes i think that sounds better.  Make sure you ask Mayra, Sue, Chris and Tom.

Joe Dix
MBi Staffing
(815) 880-5221

________________________________

**Comment by nell5318** [comment]
Yes, I agree….
